views/pages/home.pug

```diff
@@ -8,64 +8,72 @@ block content
+            img().project__card__video__clip
           .home__projects__media--card4#p4
-            video(controls muted).project__card__video__clip
+            //- video(controls muted).project__card__video__clip
+            img().project__card__video__clip
           .home__projects__media--card5#p5
-            video(controls muted).project__card__video__clip
+            //- video(controls muted).project__card__video__clip
+            img().project__card__video__clip
           .home__projects__media--card6#p6
-            video(controls muted).project__card__video__clip
+            //- video(controls muted).project__card__video__clip
+            img().project__card__video__clip
 
         .home__project__details
           .home__project__overview
             .home__project__overview--heading Overview
             .home__project__overview--description
-            .home__project__overview__links--heading Project Links
+            .home__project__overview__links--heading Links
             .home__project__overview__links--url
               include ../partials/url-icon.pug
-              a.home__project__overview__links--url--link(href="https://example.com" alt="link to live project demo" target="_blank") https://www.example.com
+              a.home__project__overview__links--url--link(href="#" alt="Link to project content" target="_blank")
 
             .home__project__overview__links--github
               include ../partials/github-icon.pug
-              a.home__project__overview__links--url--link(href="https://example.com" alt="link to live project demo" target="_blank") https://www.github.com/project-url
+              a.home__project__overview__links--url--link(href="#" alt="Project GitHub link" target="_blank")
+
+          //- TODO(alex): add this in a dynamic way
+          //- .home__project__help
+          //-   include ../partials/alxo-icon.pug
+          //-   p Explore my portfolio projects by hovering over the floating orbs.
 
           .home__project__technologies
             .home__project__technologies__header
-              .home__project__technologies__header--heading Technologies
+              .home__project__technologies__header--heading Technologies Used
             table.home__project__technologies--table
-              tr.home__project__technologies--table--row
+              tr.home__project__technologies--table--row#frontend-row
                 td Frontend
-                td.badge TypeScript
-                td.badge JavaScript
-                td.badge WebGL
-                td
-              tr.home__project__technologies--table--row
+
+              tr.home__project__technologies--table--row#backend-row
                 td Backend
-                td.badge TypeScript
-                td.badge JavaScript
-                td.badge AWS Lambda
-                td.badge DynamoDB
-                td
+
+
+      //- MOBILE VIEW
       if userDevice.isMobile
```

Comment:
Maybe need to bit the bullet on making the cards look right. or export a different set of images for the mobile view. Either way, make sure mobile's working too.